src/main.rs

```diff
@@ -82,7 +82,10 @@ enum Command {
 
     #[structopt(name = "format", about = "Format source code")]
     Format {
-        #[structopt(help = "files to format", required_unless = "stdin")]
+        #[structopt(help = "location of the project root", default_value = ".")]
```

Comment:
Rather than having the root as an option could we copy the API of Rust format, Elm format, and prettier and use the paths as relative to the current directory if they are not absolute?

I think the file IO functions in Rust may behave like this is we give them the unaltered relative path, though I have not tested

Reply:
Done.  I had other issues that lead me to believe I needed this.